src/components/Installer/index.jsx

```diff
@@ -0,0 +1,185 @@
+      isStandalone: true,
+    },
+  };
+
+  const getInstallerDetails = () => {
+    if (!os || !installerType) {
+      return null;
+    }
+    return installerData[installerType];
+  };
+
+  const installerDetails = getInstallerDetails();
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.selector_wrapper}>
+        <h2 className={styles.title}>Conda Installer Selector</h2>
+        <p className={styles.description}>
+          Choose your operating system, architecture, and installer type
+        </p>
+
+        <div className={styles.form_group}>
+          <div>
+            <label htmlFor="os-select" className={styles.label}>
+              {" "}
+              Operating System
+            </label>
+            <select
+              id="os-select"
+              value={os}
+              onChange={(e) => {
+                setOs(e.target.value);
+                setArch("");
+              }}
+              className={styles.select}
+            >
+              <option value="">Select Operating System</option>
+              {Object.keys(osArchitectures).map((osName) => (
+                <option key={osName} value={osName}>
+                  {osName}
+                </option>
+              ))}
+            </select>
+          </div>
+
+          {os && (
+            <div>
+              <label htmlFor="arch-select" className={styles.label}>
+                {" "}
+                Architecture
+              </label>
+              <select
+                id="arch-select"
+                value={arch}
+                onChange={(e) => setArch(e.target.value)}
+                className={styles.select}
+              >
+                <option value="">Select Architecture</option>
+                {osArchitectures[os].map((archOption) => (
+                  <option key={archOption} value={archOption}>
+                    {archOption}
+                  </option>
+                ))}
+              </select>
+            </div>
+          )}
+
+          <div>
+            <label htmlFor="installer-select" className={styles.label}>
+              {" "}
+              Installer Type
+            </label>
+            <select
+              id="installer-select"
+              value={installerType}
+              onChange={(e) => setInstallerType(e.target.value)}
+              className={styles.select}
+            >
+              <option value="">Select Installer Type</option>
+              {Object.entries(installerData).map(([key, data]) => (
+                <option key={key} value={key}>
+                  {data.name}
+                </option>
+              ))}
+            </select>
+          </div>
+
+          {installerDetails && (
+            <div className={styles.details_wrapper}>
+              <div>
+                <strong>Selected Configuration:</strong>
+                <ul className={styles.details_list}>
+                  <li>
+                    OS:
```

Comment:
```suggestion
                    OS:{" "}
```

Spaces can be added like this